projects/plugins/protect/src/js/routes/scan/history/status-filters.jsx

```diff
@@ -24,12 +28,14 @@ export default function StatusFilters() {
 			<ButtonGroup.Button
 				variant={ 'fixed' === filter ? 'primary' : 'secondary' }
 				onClick={ navigateOnClick( '/scan/history/fixed' ) }
+				disabled={ ! numFixed }
 			>
 				{ __( 'Fixed', 'jetpack-protect' ) }
 			</ButtonGroup.Button>
 			<ButtonGroup.Button
 				variant={ 'ignored' === filter ? 'primary' : 'secondary' }
 				onClick={ navigateOnClick( '/scan/history/ignored' ) }
+				disabled={ ! numIgnored }
```

Comment:
This is great idea but an interesting things happens when you manually visit the URL for a disabled filter (eg. `#/scan/history/ignored` when there are no ignored threats):
![Screen Shot on 2024-08-06 at 14-22-41](https://github.com/user-attachments/assets/a4f9b692-f809-4466-88cc-157cc21476a2)